### PR TITLE
New pattern: page headers

### PR DESCRIPTION
**Pull request by jtamiace**
Eyes on the truncation spec in particular would be helpful

Google doc reference https://docs.google.com/document/d/1oVVFXCOooUd_fI8Tr8YZsrL5acrDTmIBjU8kgZPkViw/edit#

### CODE REVIEW

**Review by khangmach — APPROVED** [review]

**Comment by jtamiace** [comment]
I meant to link to the button page and forgot. Will get to that

File: docs/tableOfContents.js [hunk 1]
```diff
@@ -148,6 +148,11 @@ export default [
         title: 'Pagination',
         disabled: true,
       }),
+      new Page({
+        path: '/pageheader',
+        title: 'Page header',
```
Comment (indirectlylit) [inline docs/tableOfContents.js:153]: minor note - sometimes we use singular and sometimes plural

**Review by indirectlylit — COMMENTED** [review]
This looks good.

I think it could use a bit more coordination with the [App Bar page](https://deploy-preview-144--kolibri-design-system.netlify.app/appbars/) which has some overlapping concepts (back-arrow navigation and page titles, specifically)

some examples of possible coordination:
* compare/contrast use cases
* consistent and divergent terminology as appropriate
* cross-links or even consolidation

File: docs/tableOfContents.js (hunk 1 above)
Comment (jtamiace) [reply docs/tableOfContents.js:153]: Good catch. Looks like overall, the page names seem to be plural. I can update this page and those others as plural?

Comment (indirectlylit) [reply docs/tableOfContents.js:153]: I don't have a strong preference ¯\\\_(ツ)\_/¯

**Comment by jtamiace** [comment]
@indirectlylit Does this seem accurate and help clarify the overlap between the app bar guidelines and page headers?

Screen Shot 2021-02-11 at 9 32 37 AM

**Comment by jtamiace** [comment]
...Also not really sure how to resolve this 

https://app.netlify.com/sites/kolibri-design-system/deploys/60256a0178e8e80007d503de

**Comment by nucleogenesis** [comment]
LGTM this is awesome! Unless @indirectlylit has outstanding concerns this should be good to merge

**Comment by indirectlylit** [comment]
great, thanks for addressing my feedback

**Pull request merged**